app/src/main/java/de/westnordost/streetcomplete/data/osmnotes/notequests/OsmNoteQuestController.kt

```diff
@@ -250,7 +256,7 @@ private fun Note.containsSurveyRequiredMarker(): Boolean =
     comments.any { it.containsSurveyRequiredMarker() }
 
 private fun NoteComment.containsSurveyRequiredMarker(): Boolean =
-    text?.matches(".*#surveyme.*".toRegex()) == true
+    text?.matches(".*#surveyme.*".toRegex(RegexOption.DOT_MATCHES_ALL)) == true
```

Comment:
Can we simplify this like this? I guess the regex is not really needed at all.

```suggestion
    text?.contains("#surveyme") == true
```

Reply:
I've changed the RegexOptions to also IGNORE_CASE in 3fa23fae5d8fd9a1a2865c0700cc5f5c3c8d9477. Granted, this wasn't what you asked for, but lowerCasing the text would've meant using a locale and I wanted to prevent breaking any internationalization by doing that. Would that agreeable for you?

Reply:
Yes, with the `IGNORE_CASE` option, using a regex does make sense again.

Reply:
:-)

Reply:
Well, it could also be `text?.contains("#surveyme", ignoreCase = true)`

Reply:
Well why on earth are there sooo many paths to Rome *smh* ... updated in42b7eb5a2897875d8f74bdf2c601ff361de93ae6